### ISSUE TITLE
Customizer v3

### ISSUE THREAD

**Issue by AversaConsulting** [uikit/uikit#3483]
Just wondered if there was any plans to migrate the customizer from 2.0 to 3.0? 

Cheers

**Comment by stormctf** [comment]
+1 bumping this. Any word?

**Comment by janschoenherr** [comment]
Yes, this is on our todo list. That said, I can give no time frame for this.

**Comment by gregBerthelot** [comment]
One year later, any ETA?
